app/views/task/index.html.erb

```diff
@@ -1,26 +1,47 @@
-<header><h1>Reading List</h1></header>
-
-<p class="mininav">
-    <%= link_to "Add a book", "/tasks/new" %>
-</p>
-
-<table>
-  <tr class="header">
-  <td>Title</td>
-  <td>Status</td>
-  <td>Unshelve it?</td>
-  </tr>
-
-  <% @displaytasks.each do |task| %>
-    <tr>
-      <td><%= link_to "#{task[:taskname]}", "/tasks/#{task[:id]}" %></td>
-      <td><%= task[:comp_status] %></td>
-      <td><%= link_to "Delete", "/tasks/#{task[:id]}", method: :delete, data: { confirm: "Are you sure?" } %></td>
+<header class = "header">
+  <div class="header_wrapper">
+    <h1>Unigoat Book Recs</h1>
+    <nav>
```

Comment:
I think it would be awesome to include the nav in your layout so that it's on every page rather than just on this one